<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 0, 127);</string>
    </property>
    <property name="text">
     <string>DATA ANGGOTA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>351</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Form Anggota</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>67</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID Anggota</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>77</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nama Depan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>80</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nama Tengah</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>95</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nama Belakang</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>90</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Jenis Kelamin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>47</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Telp/HP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>47</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Alamat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>47</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editID">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editNamaDepan">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editNamaTengah">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editNamaBelakang">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbJenisKelamin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Laki-Laki</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Perempuan</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="editTlp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editAlamat">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editEmail">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSimpan">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>75</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Simpan</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ICONS/add.png</normaloff>ICONS/add.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUbah">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>350</y>
       <width>75</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Ubah</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ICONS/update.png</normaloff>ICONS/update.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHapus">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>350</y>
       <width>75</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Hapus</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>ICONS/delete.png</normaloff>ICONS/delete.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="editCari">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>321</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>101</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID/Nama Anggota</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCari">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>41</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ICONS/search.png</normaloff>ICONS/search.png</iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblAnggota">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>ID Anggota</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nama Depan</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nama Tengah</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nama Belakang</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Jenis Kelamin</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Telp/HP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Alamat</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
